src/page/order-summary.page.ts

```diff
@@ -0,0 +1,9 @@
+import { $ } from 'protractor';
+
+export class OrderSummaryPage {
+  public async orderSummary(): Promise<void> {
+    await expect($('#center_column > div > p > strong').getText()).toBe(
```

Comment:
The idea with POM is just encapsulating business behavior; expects or validations should be done on the test class. Please make this method to return the confirmation text.